pages/community/index.tsx

```diff
@@ -0,0 +1,400 @@
+                          title={cardTitle}
+                          link={link}
+                          description={cardDescription}
+                        />
+                      ),
+                    )}
+                  </SimpleGrid>
+                </Container>
+              </Box>
+            ),
+          )}
+
+          <Box
+            h="1px"
+            bg="#3F3F3F"
+            opacity="0.8"
+            maxW="container.page"
+            mx="auto"
+          />
+        </Box>
+
+        <Container maxW="container.page" position="relative" py={12}>
+          <Heading as="h2" size="title.2xl" textAlign="center">
+            and most importantly...{" "}
+            <Heading
+              as="span"
+              size="title.2xl"
+              color="transparent"
+              background="linear-gradient(249.19deg, #BFA3DA 53.72%, #84309C 78.01%, #C735B0 112.02%), #ECECEC"
+              backgroundClip="text"
+            >
+              have fun!
+            </Heading>
+          </Heading>
+
+          <SimpleGrid
+            columns={{ base: 1, md: 3 }}
+            gap={{ base: 12, md: 6 }}
+            py={{ base: 12, md: 8 }}
+          >
+            {Array.from({ length: 9 }).map((_, i) => (
+              <ChakraNextImage
+                key={i}
+                src={`/assets/community/gallery/${i + 1}.png`}
+                alt="thirdweb gallery image"
+                width="350"
+                height="350"
+                objectFit="cover"
+              />
+            ))}
+          </SimpleGrid>
+        </Container>
+
+        <Center px={{ base: 4, md: 8 }} flexDir="column" mb={10}>
+          <Heading mt={8} size="display.md">
+            Start
+            <Heading
+              as="span"
+              size="display.md"
+              background="linear-gradient(248.71deg, #BFA3DA 53.44%, #84309C 72.94%, #C735B0 100.23%), #ECECEC"
+              backgroundClip="text"
+              color="transparent"
+            >
+              {" "}
+              building{" "}
+            </Heading>
+            today.
+          </Heading>
+
+          <Flex
+            my={8}
+            justify="center"
+            align="center"
+            gap={4}
+            flexDir={{ base: "column", md: "row" }}
+          >
+            <LinkButton
+              href="https://discord.gg/thirdweb"
+              size="lg"
+              onClick={() =>
+                trackEvent({
+                  category: TRACKING_CATEGORY,
+                  action: "click",
+                  title: "join-the-community",
+                })
+              }
+              background="white"
+              _hover={{
+                background: "white",
+                color: "#000",
+              }}
+              color="#000"
+              fontSize="larger"
+              leftIcon={<Icon as={BsLightningCharge} />}
+              isExternal
+              rightIcon={<></>}
```

Comment:
why? just don't render an icon instead of an empty element

Reply:
there's a `noIcon` prop you can pass

Reply:
ah I see, thanks!